ethereum/blockcreation/src/main/java/org/hyperledger/besu/ethereum/blockcreation/BlockTransactionSelector.java

```diff
@@ -467,15 +470,34 @@ private boolean transactionTooLargeForBlock(final Transaction transaction) {
   }
 
   private boolean blockOccupancyAboveThreshold() {
-    final double gasAvailable = processableBlockHeader.getGasLimit();
-    final double gasUsed = transactionSelectionResults.getCumulativeGasUsed();
-    final double occupancyRatio = gasUsed / gasAvailable;
+    final long gasAvailable = processableBlockHeader.getGasLimit();
+    final long gasUsed = transactionSelectionResults.getCumulativeGasUsed();
+    final long gasRemaining = gasAvailable - gasUsed;
+
+    final double occupancyRatio = (double) gasUsed / (double) gasAvailable;
     LOG.trace(
-        "Min block occupancy ratio {}, gas used {}, available {}, used/available {}",
+        "Min block occupancy ratio {}, gas used {}, available {}, remaining {}, used/available {}",
```

Comment:
is it confusing to log the min block occupancy ratio if we're ignoring it?

Reply:
I think we're logging the current occupancyRatio here, not the min

Reply:
I think we're logging the current occupancyRatio here, not the min

Reply:
technically we are not ignoring it, but for PoS we are setting it to be always 100%